src/main/resources/org/jenkinsci/test/acceptance/selenium/overflow.js

```diff
@@ -0,0 +1,13 @@
+var css = '* { overflow: visible !important }',
+    head = document.head || document.getElementsByTagName('head')[0],
+    style = document.createElement('style');
+
+style.type = 'text/css';
+style.id = 'ath-overflow-override'
+if (style.styleSheet){
+  style.styleSheet.cssText = css;
+} else {
+  style.appendChild(document.createTextNode(css));
+}
+
+head.appendChild(style);
```

Comment:
The order is important. In this case would be recommendable to add it at the end of the rest of CSS files.